doc/sphinx-guides/source/api/getting-started.rst

```diff
@@ -52,6 +52,20 @@ If you ever want to check an environment variable, you can "echo" it like this:
 
   echo $SERVER_URL
 
+With curl version 7.56.0 and higher, it is recommended to use --from-string with outer quote rather than -F flag without outer quote.
```

Comment:
```suggestion
With curl version 7.56.0 and higher, it is recommended to use --form-string with outer quote rather than -F flag without outer quote.
```